score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java

```diff
@@ -62,6 +63,16 @@ public CompilationArtifact compile(String filePath, List<String> dependencies) t
         Set<SlangSource> depsSources = new HashSet<>();
         File file = new File(filePath);
         Validate.isTrue(file.isFile(), "File: " + file.getName() + " was not found");
+        boolean validFileExtension = false;
+
+        Collection<File> slangFiles = FileUtils.listFiles(new File(file.getParent()), SLANG_FILE_EXTENSIONS, false);
+        for (File slangFile : slangFiles) {
+            if (StringUtils.equals(slangFile.getAbsolutePath(), file.getAbsolutePath())){
+                validFileExtension = true;
```

Comment:
I was suggested to use FileUtils.listFiles() which gets a collection of files with a specified list of extensions from a directory. In our case we specify a file, so I execute listFiles() for the file's parent directory. Afterwards I have to check that my file is among the list of files returned by the method.